### PR TITLE
setting pitch date

### PR DESCRIPTION
**Pull request by casey-chow**

### CODE REVIEW

File: client/components/PitchDateSelector/PitchDateSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+const PitchDateSelector = withAC(
+  ({
+    companyId,
+    applicationContainer: ac,
+    showText,
+    hideText,
+    selectedText,
+  }: PitchDateSelectorProps & ApplicationContainerProps) => {
+    const target = useRef(null);
```
Comment (VinaiRachakonda) [inline client/components/PitchDateSelector/PitchDateSelector.tsx:67]: Seeing this error when moving the card to the pitching state. 

Screen Shot 2019-04-14 at 9 40 28 PM

**Comment by VinaiRachakonda** [comment]
Company view should also be updated with the pitch date
Screen Shot 2019-04-16 at 2 26 15 AM

**Review by VinaiRachakonda — COMMENTED** [review]
> Company view should also be updated with the pitch date
> Screen Shot 2019-04-16 at 2 26 15 AM

Referring to the pitch date tag under the archive button

File: client/components/PitchDateSelector/PitchDateSelector.tsx (hunk 1 above)
Comment (casey-chow) [reply client/components/PitchDateSelector/PitchDateSelector.tsx:67]: I'll assume this is resolved.

**Comment by casey-chow** [comment]
Addressed, PTAL

**Review by VinaiRachakonda — APPROVED** [review]

**Pull request merged**